Automattic/wp-calypso
# Fix: Deselect Reader menu dropdown sections in navigation

#### Changes proposed in this Pull Request

Deselect the "Followed Sites", "Lists" and "Tags" dropdown sections. Also, to reduce scope, the styles applied in this PR only affect the Reader navigation. 

#### Testing instructions



* Go to `/read/conversations` using the generated Calypso live link from the comment below
* Click on "Lists", "Followed Sites" and "Tags"
* The  dropdown sections should not be selected
![navigation](https://user-images.githubusercontent.com/995955/110661731-84ba1b00-81cd-11eb-850c-92773ff6e40a.png)


Fixes #50705
